model/import/Parser/ParserInterface.php

```diff
@@ -23,15 +23,13 @@
 namespace oat\taoQtiItem\model\import\Parser;
 
 use oat\oatbox\filesystem\File;
-use oat\taoQtiItem\model\import\ItemInterface;
+use oat\taoQtiItem\model\import\CvsItemResult;
 use oat\taoQtiItem\model\import\TemplateInterface;
 
 interface ParserInterface
 {
     /**
-     * @return ItemInterface[]
-     *
      * @throws InvalidImportException
      */
-    public function parseFile(File $file, TemplateInterface $template): array;
+    public function parseFile(File $file, TemplateInterface $template): CvsItemResult;
```

Comment:
This parser interface is agnostic from CSV, could we have a `ItemImportResult` object instead?

We might have tomorrow other type of import and then CSV is too coupled to the implementation.